library/plugins/base/publisher/src/main/java/com/cdsap/talaiot/publisher/base/timeline/TimelinePublisher.kt

```diff
@@ -1,4 +1,4 @@
-package com.cdsap.talaiot.publisher.timeline
+package com.cdsap.talaiot.publisher.base.timeline
```

Comment:
I'm not sure if aiming to give compatibly is better to left the namespace as publisher.timeline.
What do you think?

Reply:
I think it is okay for base to use `publisher.timeline`. :+1:

Reply:
Actually, I'm having a second thought now. 

In my understanding, the `talaiot` module uses the "original" namespacing for legacy compatibility, so maybe we should stick with `publisher.base` namespace for the components inside of `base`.

One question. I thought that the `talaiot` module introduced in https://github.com/cdsap/Talaiot/pull/216 was supposed to be called `talaiot-legacy` according to the documentation? That, in my opinion, would make it clear that the module is for legacy compatibility.

Reply:
I reverted the change to lay in `talaiot.publisher.timeline`, the reason is because still from plugins like legacy we depend on base publisher(you will see more cases once we create more plugins/publisher).
If you are using Talaiot already and you upload to 1.4 you should be able to have working publishers with the same namespace. 

You're completely right, the name is Talaiot-legacy. 
I will change it